llvm/include/llvm/IR/Cheri.h

```diff
@@ -55,6 +56,68 @@ inline bool isCheriPointer(Type *Ty, const DataLayout *DL) {
 }
 
 namespace cheri {
+
+/// Represents the LLVM-level attribute that is used to signal
+/// cross-compartment imports.
```

Comment:
This isn’t used for imported functions, please can you be explicit about exactly what it is for?

Reply:
7d7eb43a34a29244400c478249d34f9503aaa1ca renames `CapabilityImportAttr` to `GlobalCapabilityImportAttr`. I think it's not perfect yet, in the sense that a _global_ (value) in general could refer to a function as well. Perhaps there's a better name; `GlobalVariableCapabilityImportAttr` felt a bit too long. The commit also adds a [small bit](https://github.com/CHERIoT-Platform/llvm-project/pull/147/commits/7d7eb43a34a29244400c478249d34f9503aaa1ca#diff-9bdc401a1ee7793838a466dbec73b4cd8ce3a57e222b19e06fc9d429d2c561c3R60-R61) to the documentation quoted in this comment.